.github/workflows/ci.yml

```
@@ -335,12 +335,29 @@ jobs:
           XWIN_ARCH: "x86_64"
           XWIN_CACHE_DIR: "${{ github.workspace }}/.xwin"
 
-      - name: "Build"
+  # Separate jobs for the nightly crate
+  windows-trampoline-test:
+    needs: determine_changes
+    if: ${{ github.repository == 'astral-sh/uv' && (needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main') }}
+    runs-on: windows-latest
+    name: "test windows trampoline | ${{ matrix.target-arch }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        target-arch: ["x86_64", "i686", "aarch64"]
```

```suggestion
        target-arch: ["x86_64", "i686"]
```

I don't think aarch64 will run on `windows-latest` runner